client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -882,9 +895,11 @@ class EmailProvidersComparison extends Component {
 					/>
 				) }
 
+				{ shouldPromoteGoogleWorkspace && this.renderGoogleCard() }
+
 				{ this.renderTitanCard() }
 
-				{ this.renderGoogleCard() }
+				{ ! shouldPromoteGoogleWorkspace && this.renderGoogleCard() }
```

Comment:
We are not going to switcg the order of email providers this time, we can remove all those changes.